src/main/java/com/microsoft/aad/adal4j/AdalAuthorizatonGrant.java

```diff
@@ -62,6 +62,12 @@
         if (this.params != null) {
             outParams.putAll(this.params);
         }
+        
+        if(!outParams.containsKey("scope"))
+        {
+        	outParams.put("scope", "openid");

```

Comment:
If we don't allow scope to be set, why are you checking to see if it is set in this code path.  You only set scope to openid if it wasn't already set.  I would have assumed that there should be no way that it would have been set at this point.  And if it was, then we need to figure out what to do about that.  Dropping the openid value is not an obvious course of action to me.